app/src/main/java/edu/gatech/gtorg/gitmad/threads/fragment/ThreadDetailsFragment.java

```diff
@@ -84,20 +84,24 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         downloadButton = (Button) rootView.findViewById(R.id.button_download);
         imageView = (ImageView) rootView.findViewById(R.id.download_view);
 
+        /* TODO 10
+         *  Create a new View.OnClickListener() for sendButton by using an anonymous inner class
+         *  or a lambda expression. This anonymous inner class needs to override the onClick() method.
+
+         *  Inside the onClick method, write code to tell your friends about the threads
+         *  you're interested in! You don't know how the user wants to send their message,
+         *  so use an implicit intent for sending messages.
+
```

Comment:
You may want to tell them to set and create the onClickListener in the same step.